includes/class-wc-stripe-helper.php

```diff
@@ -443,6 +443,28 @@ public static function get_order_by_setup_intent_id( $intent_id ) {
 		return false;
 	}
 
+	/**
+	 * Sanizite and retrieve a shortened statement descriptor.
+	 *
+	 * The descriptor will be used as prefix and the order ID will be concatenated, acting as a suffix.
+	 *
+	 * @param string $statement_descriptor
+	 * @param WC_Order $order
+	 * @return string $statement_descriptor Final shortened statement descriptor
+	 */
+	public static function get_shortened_statement_descriptor( $statement_descriptor = '', $order = null ) {
```

Comment:
The name of this function would make me think it retrieves the shortened static prefix – given that this includes the dynamic suffix, maybe we could call this the "dynamic statement descriptor"?

Reply:
Good call, renamed it in 18cc7bb.